eng/helix/content/runtests.cmd

```diff
@@ -20,6 +20,7 @@ set DOTNET_ROOT=%DOTNET_HOME%\%$arch%
 set DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
 set DOTNET_MULTILEVEL_LOOKUP=0
 set DOTNET_CLI_HOME=%CD%\home%random%
+set VSTEST_DUMP_PATH=%HELIX_DUMP_FOLDER%
```

Comment:
To confirm: `dotnet test` uses this environment variable for created dumps automatically❔

Reply:
Yeah they added support to control where they directly output dumps, so this makes it so we don't have to move them to the helix upload root manually